app/src/main/java/com/example/pygmyhippo/common/Event.java

```diff
@@ -89,6 +90,59 @@ public Event(String eventTitle, String eventID, String organiserID, ArrayList<En
         this.enableGeolocation = enableGeolocation;
     }
 
+    /**
+     * This method will generate the hashcode of the Event.
+     * returns boolean of if successful
+     *
+     * @return boolean
+     */
+    public boolean generateHashcode(){
```

Comment:
the function name doesn't really make sense to me, generateHashcode implies that a hashcode would be returned instead of a boolean, maybe isHashcodeCreated?

Reply:
updated